WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -45,9 +45,23 @@ final class ShippingLabelPackageDetailsViewModel: ObservableObject {
     ///
     @Published private(set) var itemsRows: [ItemToFulfillRow] = []
 
-    /// The id of the selected package, if any
+    /// The id of the selected package. Defaults to last selected package, if any.
     ///
     @Published var selectedPackageID: String?
+
+    /// The title of the selected package, if any.
+    ///
+    var selectedPackageName: String {
```

Comment:
Not really specific to this PR, but I found a bit surprising to have two `selected` properties for the custom and predefined packages, since technically the model could get in a state where both are populated. It would be nice to have a base `ShippingLabelPackage` prototype for both types to simplify the logic a bit.

Reply:
Agreed! I wanted to do that to simplify the logic for this, but it ended up being a little too much to do all at once. It would be nice to do that, though.